.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "buildroot"]
 	path = buildroot
-	url = git://git.buildroot.net/buildroot
+	url = https://github.com/buildroot/buildroot.git
```

Comment:
Why do we need https:// instead of git://?  Firewall rules?

Reply:
When using the buildroot.net hosted repo, I saw frequent timeouts and connectivity issues. The github-hosted mirror seems to be a lot more stable and seems to be kept up-to-date.

Reply:
👍